readme.md

```diff
@@ -21,6 +21,7 @@ Add the middleware just before [graphql-server](https://github.com/apollographql
 - `maxFieldSize` (integer): Max allowed non-file multipart form field size in bytes; enough for your queries (default: 1 MB).
 - `maxFileSize` (integer): Max allowed file size in bytes (default: Infinity).
 - `maxFiles` (integer): Max allowed number of files (default: Infinity).
+- `errorHandler` (function): This function is attached to the `error` event of each file stream. (default: () => {})).
```

Comment:
Maybe we don't need to expose this as an option. It's not very intuitive to users why the option exists since we document people should have an error listener on file streams in resolvers; they might wonder when should which be used, and why. Also, other than perhaps logging, I can't think of a good use case?

Reply:
Ya, the only real case here is logging. I don't mind getting rid of it, or changing the name to something like "errorLogger" or "onError" so it's less confusing.

Reply:
I vote removing it; we have the diff for how to add it later as an enhancement if people really want it.